packages/manager/modules/billing/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "globals": {
+    "moment": true
```

Comment:
since billing module main entry is es6 code (using import)
i think ```import moment from moment;``` and having moment as a dependency should be fine

Reply:
Actually, it doesn't seem used